extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/HibernateValidatorProcessor.java

```diff
@@ -345,11 +345,28 @@ public void build(HibernateValidatorRecorder recorder, RecorderContext recorderC
 
     @BuildStep
     NativeImageConfigBuildItem nativeImageConfig() {
-        return NativeImageConfigBuildItem.builder()
-                .addResourceBundle(AbstractMessageInterpolator.DEFAULT_VALIDATION_MESSAGES)
-                .addResourceBundle(AbstractMessageInterpolator.USER_VALIDATION_MESSAGES)
-                .addResourceBundle(AbstractMessageInterpolator.CONTRIBUTOR_VALIDATION_MESSAGES)
-                .build();
+        List<String> potentialHibernateValidatorResourceBundles = List.of(
+                AbstractMessageInterpolator.DEFAULT_VALIDATION_MESSAGES,
+                AbstractMessageInterpolator.USER_VALIDATION_MESSAGES,
+                AbstractMessageInterpolator.CONTRIBUTOR_VALIDATION_MESSAGES);
```

Comment:
I kept the list here as there's really no need to keep it static if we are not building a native image.

Reply:
That works. You can also use `String[]` and save some memory if you're only iterating.